Web Development/Color_Code_Converter/readme.md

```diff
@@ -0,0 +1,30 @@
+# Color Code Converter
+
+## Overview
+
+A color code converter that converts between RGB, Hex, CMYK & Hsl.
+
+## Tech Stack
+
+- [x] HTML
+- [x] CSS
+- [x] JS
+
+## Quick Start
+
+* Clone this repository
+```bash
+git clone https://github.com/abhijeet007rocks8/Dev-Scripts.git
+```
+
+* Change directory
+```bash
+cd Web_Development/Color_Code_Converter/
+```
+
+* Double Click `index.html` to open it in the browser and use the app.
+
+## Explanation Video
+
+https://user-images.githubusercontent.com/64689436/157420681-7f1e5f5b-3e11-43ec-a3ff-99f657b03c5e.mp4

```

Comment:
i think so. wrong demonstration video added :

- PR for Color Code Convertor
- Demo video added for Vector calculation.